### PR TITLE
Fix daily build

### PR DESCRIPTION
**Pull request by mandarjog**
`make istio-archive` and `make generate_yaml` both work now.

### CODE REVIEW

File: install/updateVersion.sh [hunk 1]
```diff
@@ -25,16 +25,36 @@ cd ${ROOT}
 # just run the old version
 ${ROOT}/install/updateVersion_orig.sh "$@"
 
+echo "$@"
+
+DEST_DIR=${ROOT}
+while getopts :n:d:a: arg; do
```
Comment (rkpagadala) [inline install/updateVersion.sh]: what does n argument do?

**Comment by ldemailly** [comment]
food for thought: can we setup up something so some PR can request to check / create a on demand build to test/confirm they wouldn't break the nightly process ? e.g. #4639

**Comment by ldemailly** [comment]
also, how did you test this/how do you know it fixes things?

**Comment by mandarjog** [comment]
I tested that by confirming `make istio-archive` creates an archive with the correct istio.yaml files.

**Comment by mandarjog** [comment]
@rkpagadala is it possible to request a daily build off this PR ?

**Comment by hklai** [comment]
@ldemailly yes adding some tests to prevent release breakage is on the todo list, but so are a thousand other things.  =(

@mandarjog please submit this and we can kick off a manual daily release.

**Comment by mandarjog** [comment]
@hklai can we kick off a build now ?

**Comment by hklai** [comment]
Will have to wait for https://prow.istio.io/?job=green-build to pick up your commit.

**Comment by rkpagadala** [comment]
@mandarjog the build is fixed. Thanks.

File: install/updateVersion.sh (hunk 1 above)
Comment (mandarjog) [reply install/updateVersion.sh]: Removal of this flag broke the script :-( 
`":d:a"`  does not parse a which was critical...

**Comment by ldemailly** [comment]
so it wasn't, really ?

also how can we make sure the qualification process finds the same issue found by
https://ibmcloud-perf.istio.io/regpatrol/

(which is that mixer didn't even come up)

**Pull request merged**